locales/en/kafka.json

```diff
@@ -109,6 +109,13 @@
   "partitions": "Partitions",
   "partitions_value": "up to 500",
   "select_filter_category": "Select filter category",
+  "settings": {
+    "connection_re_authentication_label": "Connection re-authentication:",
+    "disable": "Disable",
+    "kafka_instance_settings_label": "Configure global settings for the kafka instance",
+    "warning_description": "There are some potential risks",
```

Comment:
After chatting with the CCS team, here is an updated message to use:
> There are potential risks associated with turning off re-authentication, where bad actors could stay connected indefinitely.

Please note:
- There is an outstanding question in the [documentation PR](https://github.com/redhat-developer/app-services-guides/pull/553/files#r933265729) regarding whether we should use something other than "bad actor". Any decision made in that PR should ultimately be applied to this message, too. This could be a future PR, but also possible that this will be resolved while this PR is open.
- FYI, in the future, when the documentation is published, we will want to add a link "Learn more" to access the documentation. This will be covered as part of a follow-up task: https://issues.redhat.com/browse/MGDSTRM-8809

Reply:
FYI, the documentation PR mentioned above now uses "attacker" instead of "bad actor". Following their updates, we would use this for our message instead of the version in my previous comment:
> There are potential risks associated with turning off re-authentication, where attackers could stay connected indefinitely.